src/main/resources/org/boozallen/plugins/jte/TemplateEntryPoint.groovy

```diff
@@ -63,6 +64,12 @@ void createWorkspaceStash(){
   }catch(any){}
 }
 
+void archiveConfig(){
+    node{
+        writeYaml data: pipelineConfig, file: "aggregated_config.yaml"
```

Comment:
Thinking out loud ... how about adding a new variable for the actual pipelineConfig to prevent the breakage?  not sure what we would name the new var, (maybe templateConfigObject) and it might be confusing so maybe changing it to the actual TemplateConfigObject is better

Reply:
will change to writeFile using the serialize

Reply:
added templateConfigObject var to script containing the final config